README.md

```diff
@@ -1,31 +1,207 @@
-# Neon Diana
-Device Independent API for Neon Applications (Diana) is a collection of microservices that provide various functionality 
-to NeonAI systems. All services are implemented as standalone Docker containers.
+# Install and Deploy NeonAI Diana 
 
-Install the Diana utilities Python package with: `pip install neon-diana-utils`
-The `diana` entrypoint is available to handle automated setup and some common administration tasks. You can see get detailed
-help via `diana --help`.
+Device Independent API for Neon Applications (Diana) is a collection of microservices that add functionality to NeonAI systems. Diana microservices are deployed in a Kubernetes cluster.
 
-## Backend
-The Neon Backend is a collection of services that are provided to client applications.
-Some backend services connect via a RabbitMQ server, while others are served via HTTP.
+After you have installed Diana, use `diana --help` to get detailed help.
 
-### Prerequisites
-- `python3` is required to install `neon-diana-utils`
-- `kubectl` and `helm` are needed to apply the generated configurations
+For more information about using NeonAI, see [the NeonAI documentation site](https://neongeckocom.github.io/neon-docs/).
 
-### Configure Backend with `diana`
-A Diana backend can be configured automatically with `diana configure-backend`. A standard example is included here, but 
-a description of config options is available via: `diana configure-backend --help`.
+## Prerequisites
 
-```shell
+* [Ubuntu](https://ubuntu.com/) 20.04 or later. 
+  * Diana will most likely work on other flavors of Linux, but we have not yet verified this.
+* [Python](https://www.python.org/getit/) 3.8 or later.
+* [Pip](https://pypi.org/project/pip/) Python package installer.
+* [Kubectl](https://kubernetes.io/docs/reference/kubectl/) Kubernetes command-line tool.
+* [Helm](https://helm.sh/) package manager for Kubernetes.
+* A [Kubernetes](https://kubernetes.io/) installation.
+  * The following instructions assume a local installation using [Microk8s](https://microk8s.io/) version 1.26/stable or later.
+  * You can likely deploy Diana on a Kubernetes cluster in the cloud, but we have not yet verified this.
+* [Neon Skill Utilities](https://github.com/NeonGeckoCom/neon-utils) 1.0 or later.
```

Comment:
This shouldn't be necessary to install separately; was there an with with dependencies after installing just diana?

Reply:
You shouldn't have to touch `neon-utils` at all; if you do then its something I should fix in `neon-diana-utils`